tests/concepts/test_View.cpp

```diff
@@ -0,0 +1,28 @@
+#include "gtest/gtest.h"
+
+#include "kokkos-utils/concepts/View.hpp"
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+/**
+ * @file
+ *
```

Comment:
It seems you're missing the `addtogroup` ?

Reply:
Indeed, added it.